src/main/kotlin/app/Main.kt

```diff
@@ -61,9 +65,23 @@ fun main(args: Array<String>) {
             }
         }
 
+        // Keep track of the rate limit broadcast actions associated with each socket to easily cancel on close
```

Comment:
Should this be moved elsewhere (e.g. a socket broadcast manager)?

Reply:
Yes, sounds a lot cleaner